Makefile

```diff
@@ -193,21 +193,32 @@ TARGETS = \
 	help \
 	info \
 	clean \
-	electron-develop
+	electron-develop \
+	publish-aws
 
 ifeq ($(TARGET_PLATFORM),darwin)
 electron-installer-app-zip: release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).zip
 electron-installer-dmg: release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).dmg
+publish-aws: release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).dmg
```

Comment:
The ZIP is currently being uploaded to S3 (since like v1.0.0-beta.4), but its not promoted anywhere, since its only meant to be used by the auto-update engine (which requires a ZIP). We never got the auto-update feature up and running, so the ZIP is unused at the moment, but its still there.